### ISSUE TITLE
Try to remove all compiler warnings

### ISSUE THREAD

**Issue by SoniEx2** [NOVA-Team/NOVA-Core#184]
Please try to remove all compiler warnings. Either with `@SuppressWarnings` (e.g. generics in MC code) or otherwise.

**Comment by Caellian** [comment]
Yeah, that would be neat, but keep in mind that it would also make code
harder to read. It is generally a good idea to do such a thing after mojor
coding is done because it can/will cause unclean code and ocd seizures.
